File: autopilot/v1/task/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

task = @client.autopilot.assistants('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .tasks('UDXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .update(actions: {
       'actions' => [
           {
               'say' => {
                   'speech' => 'I was going to look for my missing watch, but I could never find the time.'
               }
           },
       ]
   })

puts task.friendly_name

File: autopilot/v1/defaults/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Autopilot.V1.Assistant;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var defaults = new Dictionary<string, Object>()
        {
            {"defaults", new Dictionary<string, Object>()
                {
                    {"assistant_initiation", "task://hello-world"},
                    {"fallback", "task://hello-world"}
                }}
        };

        var defaults = DefaultsResource.Update(
            defaults: defaults,
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(defaults.AssistantSid);
    }
}
